MANIFEST.in

```diff
@@ -1,12 +1,16 @@
-prune clients
-prune tests
```

Comment:
I'm not certain, but I think certain files will automatically be added to the package unless you explicitly prune them.  I would keep the `prune tests` line.